src/NLog/LayoutRenderers/NdcLayoutRenderer.cs

```diff
@@ -81,7 +81,7 @@ protected override void Append(StringBuilder builder, LogEventInfo logEvent)
         {
             if (TopFrames == 1)
             {
-                // Allows fast rendering of ${when:when='${ndc:topframes=1}' == '':inner=:else=${ndc}|}
+                // Allows fast rendering of ${ndc:topframes=1}
```

Comment:
:+1: that's a easier example :)

Reply:
SonarLint complained about code in comments. Trying to get around that.

Reply:
another way is to move it to xml docs, e.g. `<example>` or `<remarks>`